rust/types/src/collection.rs

```diff
@@ -222,6 +229,14 @@ impl TryFrom<chroma_proto::Collection> for Collection {
             },
             None => None,
         };
+        let updated_at =
+            proto_collection
+                .updated_at
+                .ok_or(CollectionConversionError::MissingField(
```

Comment:
Will this fail queries during upgrade for e.g. if query service is on a newer version with this change and sysdb is on an older version without this change?

Reply:
yes, updated